xl-op/digitalai/dai-release/kubernetes-generic/dai-release_cr_default.yaml.tmpl

```diff
@@ -101,7 +101,7 @@ spec:
       nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
       nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
       nginx.ingress.kubernetes.io/rewrite-target: /
-      nginx.ingress.kubernetes.io/session-cookie-name: JSESSIONID
+      nginx.ingress.kubernetes.io/session-cookie-name: ROUTE
```

Comment:
Same comment as for deploy:
Please check if that is happening for the haproxy, I think we need there change that config and for external too, check lines 129 and 141. Do we need the same change on the openshift too?